### PR TITLE
[Docs] Add the new available_time_ranges dashboard config setting to the docs

### PR DESCRIPTION
**Pull request by AdityaHegde**
Since we added `available_time_ranges`, updating the docs to add to the dashboard config doc page. Also adding a page mentioning the rill extensions.

### CODE REVIEW

File: docs/docs/reference/rill-iso-extensions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+| Rill ISO extension | Description      | As Time Range | As Time Comparison |
+|--------------------|------------------|---------------|--------------------|
+| inf                | All time         | Yes           | No                 |
+| rill-TD            | Today            | Yes           | No                 |
+| rill-WTD           | Week to Date     | Yes           | No                 |
+| rill-MTD           | Month to Date    | Yes           | No                 |
+| rill-QTD           | Quarter to Date  | Yes           | No                 |
+| rill-YTD           | Year to Date     | Yes           | No                 |
+| rill-YTD           | Year to Date     | Yes           | No                 |
+| rill-PP            | Last Period      | No            | Yes                |
```
Comment (nishantmonu51) [inline docs/docs/reference/rill-iso-extensions.md:21]: should we call this Previous Period instead of Last Period?

Comment (AdityaHegde) [reply docs/docs/reference/rill-iso-extensions.md:21]: Hmmm the label in UI says `Last Period` So we can either switch to `rill-LP` and `Last Period` for label. Or `rill-PP` and `Previous Period`

Comment (nishantmonu51) [reply docs/docs/reference/rill-iso-extensions.md:21]: ok, then lets keep the current one.

**Comment by AdityaHegde** [comment]
Will raise a separate PR to update the UI label and the docs to `Previous Period`

**Pull request merged**